iwyu.cc

```diff
@@ -2951,7 +2953,11 @@ class InstantiatedTemplateVisitor
   }
 
   void ReportTypeUse(SourceLocation used_loc, const Type* type,
-                     const char* comment = nullptr) override {
+                     const char* comment = nullptr,
+                     const set<const Type*>& blocked_types = {}) override {
+    // 'blocked_types' argument can be filled only in the base class
+    // implementation.
+    CHECK_(blocked_types.empty());
```

Comment:
This feels like another indication that `blocked_types` is not really an argument to `ReportTypeUse`.